src/liboslexec/shadingsys.cpp

```diff
@@ -405,6 +405,48 @@ ShadingSystem::query_closure (const char **name, int *id,
 
 
 
+SGBits
+ShadingSystem::globals_bit (ustring name)
+{
+    // **MUST** match the order of SGBits!!
```

Comment:
I'm not a big fan of these cases where things need to stay in sync. Since its a plain linear search in the end, why not set it up as:

`pair<ustring, SGBit> globals[]`

So that you can list the name and corresponding `SGBit` together (both functions can share the same table)